_posts/2023-02-23-zennews-summarization.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+author: "Baris Can Durak"
+title: "ZenNews: Generate summarized news on a schedule"
+description: "A tool powered by ZenML that can automate the summarization of news sources and save users time and effort while providing them with the information they need.."
+category: zenml
+tags: tooling zenml summarization news
+publish_date: February 23, 2023
+date: 2023-02-23T00:02:00Z
+thumbnail: /assets/posts/zennews/zennews.jpeg
+image:
+  path: /assets/posts/zennews/zennews.jpeg
+---
+
+**Last updated:** February 23, 2023
+
+![](/assets/posts/zennews/zennews.jpeg)
+*Image generated by [StableDiffusion 2.1](https://huggingface.co/stabilityai/stable-diffusion-2-1)*
```

Comment:
```suggestion
![*Image generated by [StableDiffusion 2.1](https://huggingface.co/stabilityai/stable-diffusion-2-1)*](/assets/posts/zennews/zennews.jpeg)
```

I *think* this syntax should also work. This way we're also providing the image caption for accessibility.